pandas/core/index.py

```diff
@@ -1348,10 +1348,28 @@ def order(self, return_indexer=False, ascending=True):
             return sorted_index, _as
         else:
             return sorted_index
+    order = sort_values
```

Comment:
also put a PendingDeprecationWarning in order here? (so it is consistent with Series.order)